src/ResizeSensor.js

```diff
@@ -98,11 +98,15 @@
          * @returns {String|Number}
          */
         function getComputedStyle(element, prop) {
+            var computedElementStyle;
+            
             if (element.currentStyle) {
                 return element.currentStyle[prop];
             }
-            if (window.getComputedStyle) {
-                return window.getComputedStyle(element, null).getPropertyValue(prop);
+            
+            computedElementStyle = window.getComputedStyle(element, null);
+            if (window.getComputedStyle && computedElementStyle) {
```

Comment:
if it is false, then the code inside the condition just won't be executed, right?